src/osemgrep/targeting/dune

```diff
@@ -0,0 +1,17 @@
+; Various operations having to do with selection of files in Semgrep.
+; What they have in common is:
+; - dependency on utilities to manipulate file paths (e.g. globbing);
+; - no dependency on full-fledged language parsing or the resulting ASTs.
+;
+(library
+  (name osemgrep_targeting)
```

Comment:
can you add (wrapped false) to be consistent
with the other libraries.

Reply:
I'm fine with this practice **only for application code that doesn't get distributed as libraries**. I can't guarantee that the modules being created today have unique names in the OCaml world, and since OCaml hasn't been blessed with namespaces, this creates incompatibilities with other libraries.